src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -155,6 +154,23 @@ static boolean isCdf(CVRSource source) {
     return source.getProvider() != null && source.getProvider().toUpperCase().equals(CDF_PROVIDER);
   }
 
+  private static boolean candidateStringMatchesLabel(
```

Comment:
Please add a line or two comment to describe what this does. The parameter names are a bit confusing.